src/ast/mod.rs

```diff
@@ -5818,6 +5818,13 @@ pub enum SqlOption {
         range_direction: Option<PartitionRangeDirection>,
         for_values: Vec<Expr>,
     },
+    /// MS SQL Server specific: Optional parameters of identity column
+    /// E.g.
+    ///
+    ///   IDENTITY(1, 2)
+    ///
+    /// [MS SQL Server]: https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql-identity-property
+    Identity { seed: Value, increment: Value },
```

Comment:
One thing regarding the types, if the seed or increment can be a negative number we would want to represent it as an expression because the Value type isn't able to represent those - see [parse_number()](https://github.com/sqlparser-rs/sqlparser-rs/blob/04a53e57538524a9cda8ca7304312113849aec52/src/parser/mod.rs#L7454)